tests/rptest/redpanda_cloud_tests/high_throughput_test.py

```diff
@@ -478,6 +478,8 @@ def get_broker_pod(self):
 
     @cluster(num_nodes=2)
     def test_throughput_simple(self):
+        assert self.redpanda.cluster_healthy(
```

Comment:
These need to go in one place which will be called automatically before and after the test. Maybe the "before" check in `make_redpanda_service_cloud`? and the after check in `tearDown`?

Reply:
since the `make_redpanda_service_cloud()` looks to be called in just the `__init__()` method, i think a more suitable place would be `setup()` which is called before every `test_*` method and failures in `setup()` will fail the test.

it looks like ducktape won't fail a test if there is a failure in the `tearDown()` method, however. it will just log a warning: https://github.com/redpanda-data/ducktape/blob/62e0285f6b3a2f22fd4a43b5fdbc13be8d4290d9/ducktape/tests/runner_client.py#L295

so, i'll leave it at the end of each individual test unless there is a more suitable place.

Reply:
> it looks like ducktape won't fail a test if there is a failure in the `tearDown()` method, however. it will just log a warning:

Ah, that explains something that John had written in our variation of the @cluster decorator: they use it exactly to avoid this problem with tearDown.

So let's do this for now but we can switch to a decorator soon enough.